packages/mobile/src/screens/personal-info/components/personal-info-form/moke-data.ts

```diff
@@ -0,0 +1,39 @@
+export const TEST_USER = {
```

Comment:
Let's create a separate src/mock folder so we won't miss it later

Reply:
corrected